Uchu.World/Handlers/GameMessages/InventoryHandler.cs

```diff
@@ -5,6 +5,19 @@ namespace Uchu.World.Handlers.GameMessages
 {
     public class InventoryHandler : HandlerGroup
     {
+        [PacketHandler]
+        public async void RequestMoveBetweenInventoriesHandler(RequestMoveItemBetweenInventoryTypesMessage message, Player player)
+        {
+            var inventoryManager = message.Item.Inventory.ManagerComponent;
+            if (inventoryManager.GameObject != player)
+                return;
+
+            await inventoryManager.MoveItemBetweenInventoriesAsync(message.Item, message.Count, message.Source,
+                message.Destination, showFlyingLoot: message.ShowFlyingLoot);
+
+            // TODO: show animation dropping item out of the vault (https://youtu.be/hJl8WvLz6rM?t=660)
```

Comment:
I am not sure this actually is related. Looks like the item animation is showing a different item.